utility/src/test/java/org/oppia/android/util/logging/EventBundleCreatorTest.kt

```diff
@@ -2019,7 +2019,7 @@ class EventBundleCreatorTest {
     // within the same application instance.
     @Provides
     @Singleton
-    @LearnerStudyAnalytics
+    @EnableLearnerStudyAnalytics
     fun provideLearnerStudyAnalytics(): PlatformParameterValue<Boolean> {
```

Comment:
Let’s update this provider method name, too, for consistency.

Reply:
Done